group_vars/all

```diff
@@ -26,3 +26,7 @@ data_bridge: br-data
 
 win2016_compute_nodes: "{{ groups['win2016-compute'] }}"
 devstack_ip: "{{ groups['devstack'][0] }}"
+
+eventlogPath: "{{ win_dir.event_log }}"
```

Comment:
these variables have no use, the filenames are "hardcoded" when you copy them so there is no sense to have them, also they are used in some places and not in others for example collect-event-log.ps1 line 41-48